pkg/interface/src/views/apps/chat/components/ChatPane.tsx

```diff
@@ -79,36 +101,27 @@ export function ChatPane(props: ChatPaneProps): ReactElement {
   } = props;
   const graphTimesentMap = useGraphTimesent(id);
   const ourContact = useOurContact();
-  const chatInput = useRef<NakedChatInput>();
+  const chatInput = useRef<{ uploadFiles: (files: FileList | File[]) => void }>();
+  const setMessage = useChatStore(s => s.setMessage);
 
   const onFileDrag = useCallback(
     (files: FileList | File[]) => {
       if (!chatInput.current) {
         return;
       }
-      (chatInput.current as NakedChatInput)?.uploadFiles(files);
+      chatInput.current?.uploadFiles(files);
```

Comment:
This ref technique is fine for now, but it's pervasive bc you want the drop target to be quite close to the root of the component tree but the receiver is usually quite deep in the tree. We should consider refactoring it.

Reply:
Yeah I didn't like this either was just trying to do the conversion.  Now that I'm looking at it again, I think it would be quick to refactor if y'all think we should just do it in this change set.

Reply:
Go nuts

Reply:
It'd be nice if this solution worked for our other drag and drop inputs as well